cypress/support/ContentfulApi.js

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Will the basic code to access contentful be move to the npm package?

Reply:
Not yet. I'd prefere we be conservative adding things to the npm package since once it's in there it'll be more difficult to change, and I'm not confident enough yet in my js or Cypress code to want it in a shared library.
I also think we should only add things to the package we know will be used across repos, so we don't end up bloating it with useless code. What should be added to the npm package might be a good thing to discuss in automation refinements.